draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1919,8 +1919,7 @@ CachedInformationType Values" registry that was defined in [RFC7924].
 ## Hash Algorithms    {#hash_algorithms}
 
 IANA is asked to establish a registry of hash algorithm values, named
-"Hash algorithm numbers for Certificate Transparency logs" initially
-consisting of:
+"CT Hash Algorithms" initially consisting of:
```

Comment:
There needs to be a comma before "initially".

I think "s/ initially consisting/, that initially consists/" would be better still.